docs/source/user-guide/getting-started.rst

```diff
@@ -16,16 +16,17 @@ Prerequisites
 
 Before starting the tutorials, you need to install:
 
-- `Miniconda or Anaconda <https://docs.anaconda.com/anaconda/install/>`_
+- `Miniconda <https://docs.anaconda.com/free/miniconda/>`_ or `Anaconda <https://docs.anaconda.com/free/anaconda/install/>`_
 - conda-build
 - Git
 
 The most straightforward way to do this is to install Miniconda or
 Anaconda, which contain conda, and then use conda to install conda-build
-and Git. Make sure you install these packages into a new environment
-and not your base environment.::
+and Git. Make sure you install these packages into your base environment.::
 
-    conda create -n my-conda-build-environment conda-build git
+    conda create -n base conda-build git
```

Comment:
If indeed `git` needs to be installed into the base env, then should this command be `conda install -n base conda-build git` instead?

Reply:
Oh, you're absolutely right. The environment doesn't need to be created anymore. Whoops!